pyresample/geometry.py

```diff
@@ -415,24 +422,33 @@ def _corner_is_clockwise(lon1, lat1, corner_lon, corner_lat, lon2, lat2):
         is_clockwise = -np.pi < angle < 0
         return is_clockwise
 
-    def get_edge_lonlats(self, frequency=None):
+    def get_edge_lonlats(self, vertices_per_side=None, frequency=None):
         """Get the concatenated boundary of the current swath."""
-        lons, lats = self.get_bbox_lonlats(frequency=frequency, force_clockwise=False)
+        if frequency is not None:
+            warnings.warn("The `frequency` argument is pending deprecation, use `vertices_per_side` instead",
+                          PendingDeprecationWarning, stacklevel=2)
+        vertices_per_side = vertices_per_side or frequency
+        lons, lats = self.get_bbox_lonlats(vertices_per_side=vertices_per_side, force_clockwise=False)
         blons = np.ma.concatenate(lons)
         blats = np.ma.concatenate(lats)
         return blons, blats
 
-    def get_edge_bbox_in_projection_coordinates(self, frequency: Optional[int] = None):
+    def get_edge_bbox_in_projection_coordinates(self, vertices_per_side: Optional[int] = None,
+                                                frequency: Optional[int] = None):
         """Return the bounding box in projection coordinates."""
-        x, y = self._get_bbox_elements(self.get_proj_coords, frequency)
+        if frequency is not None:
+            warnings.warn("The `frequency` argument is pending deprecation, use `vertices_per_side` instead",
+                          PendingDeprecationWarning, stacklevel=2)
+        vertices_per_side = vertices_per_side or frequency
+        x, y = self._get_bbox_elements(self.get_proj_coords, vertices_per_side)
         return np.hstack(x), np.hstack(y)
 
-    def boundary(self, frequency=None, force_clockwise=False):
+    def boundary(self, vertices_per_side=None, force_clockwise=False, frequency=None):
         """Retrieve the AreaBoundary object.
 
         Parameters
         ----------
-        frequency:
+        vertices_per_side (formerly frequency):
```

Comment:
This parentheses needs to be removed too.

Reply:
Done in e817700